FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -12,7 +12,7 @@ struct HomeView: View {
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
-    private let levelItem: [LevelItem] = [
+    private let levelItem: [LevelItem] =     [
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/62d1ac1ae117a3892d1f335a1f1a2d7c62129a18/FlashEnglish/Sources/Views/Home/HomeView.swift#L15"></span>Operators should be surrounded by a single whitespace when they are being used<br />
<code>operator_usage_whitespace</code> <code>HomeView.swift:15</code></td>
    </tr>
  </tbody>
</table>